src/commands/oas2json.js

```diff
@@ -1,40 +1,87 @@
 import { Command } from 'commander'
+import filenamify from 'filenamify'
 import fs from 'fs-extra'
-import YAML from 'yaml'
+import _get from 'lodash.get'
+import _trimStart from 'lodash.trimstart'
 import path from 'path'
-import { exit } from 'process'
 import pino from 'pino'
+import { exit } from 'process'
+import YAML from 'yaml'
+
 import { fromSchema } from '../utils/openapi-schema-to-json-schema-wrapper.cjs'
 
 const COMPONENT_REF_REGEXP = /#\/components\/schemas\/[^"]+/g
```

Comment:
Is there a reason why we need to cast a regex against the stringified version of the OpenAPI files instead of using `lodash.get` to do the property check of the parsed OpenAPI def object?

Reply:
I suppose it's an easier way of finding all nested instances of the `$ref` keys and saves having to traverse the entire object which could potentially be quite large